src/main/java/dk/kudishin/simpletasklist/services/TaskProcessor.java

```diff
@@ -10,7 +10,6 @@
 import java.util.List;
 
 @Service
-@Transactional
```

Comment:
If there is no need for transactions wrapping multiple DB operations, it's better to have transactions on the level of repository/dao.